packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,567 @@
+Grid is built using the [Flex component](/utilities/flex) and is responsive. Grid column widths grow and shrink based on the parent element width.
+
+## Examples
+
+### Equal column widths
+
+Use Column to lay out equal-sized content, such as a feed of cards.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Instant Provisioning</Heading>
+      <Paragraph>The API and the Console allow you to search and provision local, toll-free, or mobile numbers by capability, geography, prefix or keyword.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Vast Inventory</Heading>
+      <Paragraph>Twilio maintains phone number inventory at a rate of 50 times the quantity provisioned daily for over 600 area codes and prefixes.</Paragraph>
+    </Card>
+  </Column>
+  <Column>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Quality Screening</Heading>
+      <Paragraph>Every phone number in the inventory passes a rigorous 60-day screening process to verify capabilities and test for unwanted call traffic.</Paragraph>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+### Unequal column widths, using column span
+
+Use the `span` prop to allow content to span multiple columns. Span calculates the width of a column based off a 12 column grid. The value of span becomes the
+numerator while 12 is the denominator, which gives us a percentage width value for each column. The default span is 1 column width. In our example we're
+setting one column to equal 8 of the 12 columns, and the other to be 4 of the 12 columns.
+
+<LivePreview scope={{Grid, Column, Card, Heading, Paragraph, Button}} language="jsx">
+  {`<Grid gutter="space30">
+  <Column span={8}>
+    <Heading as="h4" variant="heading40">Getting Started with Parse</Heading>
+    <Paragraph>
+      Use SendGrid to make your email a two-way street. The Parse Webhook is a powerful tool that can take incoming email for your domain or subdomain and direct it to your application. The possibilities of this are endless!
+    </Paragraph>
+    <Heading as="h4" variant="heading40">Your Email Funnel</Heading>
+    <Paragraph>
+      Expert Insights analyzes each layer of your email funnel to determine leakage points (such as blocked emails) and opportunities to improve deliverability and engagement.
+    </Paragraph>
+  </Column>
+  <Column span={4}>
+    <Card padding="space70">
+      <Heading as="h4" variant="heading40">Expert Insights</Heading>
+      <Paragraph>Send better email with Expert Insights. Our detailed monthly reports with expert recommendations will give you deeper visibility into the health and performance of your email program.</Paragraph>
+      <Button>Add to Plan</Button>
+    </Card>
+  </Column>
+</Grid>`}
+</LivePreview>
+
+<Callout variant="warning">
```

Comment:
This could just be the default Callout variant since it's not exactly a warning about the content, or indicating that anything's wrong